packages/context/src/interceptor-chain.ts

```diff
@@ -8,46 +8,57 @@ import {BindingFilter} from './binding-filter';
 import {BindingAddress} from './binding-key';
 import {BindingComparator} from './binding-sorter';
 import {Context} from './context';
-import {InvocationResult, Next} from './invocation';
+import {InvocationResult} from './invocation';
 import {transformValueOrPromise, ValueOrPromise} from './value-promise';
-
 const debug = debugFactory('loopback:context:interceptor-chain');
 
+/**
+ * The `next` function that can be used to  invoke next generic interceptor in
+ * the chain
+ *
+ * @typeParam T - Return type of `next()`
+ */
+export type Next<T = InvocationResult> = () => ValueOrPromise<T>;
```

Comment:
Can you leave out the default value for the generic argument `T`? I am concerned that with the default argument present, it's easy to accidentally use `Next` in places where we should use `Next<SomeOtherT>` instead.